uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -11,42 +11,32 @@ define([
   function ToilRequestCtrl ($log, $q, $rootScope, OptionGroup, AbsenceType, parentCtrl) {
     $log.debug('ToilRequestCtrl');
 
-    var vm = parentCtrl;
-
-    vm.requestCanExpire = true;
-
-    vm.calculateBalanceChange = calculateBalanceChange;
-    vm.calculateToilExpiryDate = calculateToilExpiryDate;
-    vm.checkSubmitConditions = checkSubmitConditions;
-    vm.changeInNoOfDays = changeInNoOfDays;
-    vm.clearExpiryDate = clearExpiryDate;
-    vm.initChildController = initChildController;
-    vm.loadAbsencePeriodDatesTypes = loadAbsencePeriodDatesTypes;
-    vm.updateAbsencePeriodDatesTypes = updateAbsencePeriodDatesTypes;
-    vm.updateExpiryDate = updateExpiryDate;
+    parentCtrl.requestCanExpire = true;
+
+    parentCtrl.calculateBalanceChange = calculateBalanceChange;
+    parentCtrl.calculateToilExpiryDate = calculateToilExpiryDate;
```

Comment:
why is this added to the scope of the parent ctrl? it's used only internally

Reply:
Done. I removed `controller.calculateToilExpiryDate();` function calls from tests since this function is now called in TOIL on any date setting so `setTestDates(date2016, date2016To);` plus `$digest` is enough to call it.